src/main/webapp/app/plagiarism/manage/plagiarism-split-view/plagiarism-split-view.component.scss

```
@@ -8,18 +8,88 @@
+    }
+
+    // "Show only left/right" fully collapses the other pane and hides the gutter. Implemented as a class
+    // (not via panelSizes) because p-splitter coerces a 0 size to an even 50% split, so a pane can never be
+    // hidden through panelSizes alone. The panels already set overflow:hidden / min-width:0 above.
+    &.collapsed-right {
+        .p-splitterpanel:first-child {
+            flex: 1 1 100% !important;
+        }
+        .p-splitterpanel:last-child {
+            flex: 0 0 0 !important;
+        }
```

Stylelint (17.13.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)